eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -161,25 +171,40 @@ def fitJWST(eventlabel, s4_meta=None):
                                 else:
                                     new_file.write(line)
 
-                # Set the intial fitting parameters
-                params = p.Parameters(param_file=meta.fit_par)
-
                 # Subtract off the zeroth time value to avoid floating point precision problems when fitting for t0
                 t_offset = np.floor(meta.bjdtdb[0])
                 t_mjdtdb = meta.bjdtdb - t_offset
                 params.t0.value -= t_offset
 
-                # Get the flux and error measurements for the current channel
-                flux = meta.lcdata[channel,:]
-                flux_err = meta.lcerr[channel,:]
-
-                # Normalize flux and uncertainties to avoid large flux values
-                flux_err = flux_err/ flux.mean()
-                flux = flux / flux.mean()
+                # Temporary normalization to avoid large flux values (FINDME: replace when constant offset is implemented)
+                flux = meta.lcdata[0,:] / np.mean(meta.lcdata[0,:200])
```

Comment:
I think this normalization scheme conflicts with more recent changes.  Be very careful when merging conflicts in this pull request!

Reply:
Last time I branched from main, the normalization was thrown off so I was getting crappy fits because the out-of-transit had been normalized to a mean of 1.2 or something instead of 1. I think that until we have a constant offset function in the fit we'll just have to fix this part every time we make changes to something. But I can remove this if it conflicts during the merge.

Reply:
The `PolynomialModel` class with the `c0` free parameter fits for a constant offset.

Reply:
Got it now, thanks!